client/my-sites/plans-v2/details.tsx

```diff
@@ -116,10 +137,27 @@ const DetailsPage = ( {
 	const isBundle = [ OPTIONS_JETPACK_SECURITY, OPTIONS_JETPACK_SECURITY_MONTHLY ].includes(
 		productSlug
 	);
-	const backButton = () => page( selectorPageUrl );
+	const backButton = () => {
+		dispatch(
+			recordTracksEvent( 'calypso_subtypes_back_click', {
+				site_id: siteId || undefined,
+				product_slug: productSlug,
+				duration,
+			} )
+		);
+		page( selectorPageUrl );
+	};
+
+	const viewTrackerPath = siteId
+		? `${ rootUrl }/:product/${ durationToString( duration ) }/details/:site`
+		: `${ rootUrl }/:product/${ durationToString( duration ) }/details`;
+	const viewTrackerProps = siteId
+		? { site: siteSlug, product: productSlug }
+		: { product: productSlug };
```

Comment:
If we have a site in the context, we want to track that as well as part of the page view tracking object.